src/lando/main/management/commands/start_uplift_worker.py

```diff
@@ -13,3 +16,26 @@
 class Command(StartWorkerCommand):
     help = "Start an uplift worker."
     worker_class = UpliftWorker
+
+    def setup_moz_phab_config(self):
+        """Copy the moz-phab config file to the user's home directory."""
+        source_config = settings.BASE_DIR / "api/legacy/workers/config/moz-phab-config"
+
+        home_dir = Path.home()
+        dest_config = home_dir / ".moz-phab-config"
+
+        if not source_config.exists():
+            raise CommandError(
+                f"moz-phab config file not found at {source_config}. "
+                f"The uplift worker requires this configuration file "
+                f"to be loaded at {dest_config}."
+            )
+
+        logger.info(f"Setting up moz-phab config: {source_config} -> {dest_config}.")
+        shutil.copy2(source_config, dest_config)
+        logger.info("moz-phab config installed successfully.")
+
+    def handle(self, name: str, **options):
+        """Set up moz-phab config before starting the worker."""
+        self.setup_moz_phab_config()
```

Comment:
We should probably have something that checks that the config has indeed been picked up correctly by moz-phab? Maybe a simple check for one of the values that deviate from the defaults.